electron-builder.yml

```diff
@@ -19,5 +19,10 @@ win:
     - msi
     - zip
 linux:
+  category: Audio;Player
+  desktop:
+    Keywords: bcc;bmm;electron;media;audio;christian;faith;speech;music
```

Comment:
I'd remove `electron` from this list 😅 Rest looks good to me 😎 Maybe `Brunstad` would be a good addition ...

Reply:
I included `electron` because an example I saw did it too, but I can remove it.
The main use for these, is that your shell can match these when you search for applications:

![image](https://github.com/bcc-code/bmm-web/assets/1202149/6d1a0550-53de-42a6-8dd9-43284ef5f189)

> **Note**
> This example uses a different BMM.desktop file that I wrote myself to integrate an older release. The icon and text doesn't match the desktop file generated by this branch, but it demonstrates the point.

Reply:
Its fine IMO. Just find it a bit daunting when apps appear in a search which seemingly have no relation to the keyword ... I wouldn't expect anything built with `java` to appear when I acutally just want to start something having `java` in its name, but that's me ... 🤗

Reply:
Changed the keyword list to `audio;bcc;bmm;brunstad;christian;church;edification;faith;media;music;sermon`

- Removed: _electron_
- Added: _edification_
- Replaced: _speech_ which is more common for TTS / speech synthesis whit _sermon_
- Sorted alphabetically